### DISCUSSION TITLE
[Help]: Fullwidth loop carousel

### DISCUSSION THREAD

**Discussion by zlateska** [davidjerleke/embla-carousel#852] [Help]
### Summary

Hi,

I'm working on a fullwidth carousel where four active items align perfectly with the container's width, while the additional items are meant to overflow outside the visible area. I've run into a hiccup with the looping behavior, however.

The issue is that items are shifted from the start to the end or vice versa too early during their loop transitions. This early movement causes the items to disappear or reappear, disrupting the seamless loop effect.

I'm looking for advice on how to tweak the looping logic so that the transition of items is delayed until they are completely outside the visible area.

Here’s a screen recording:
https://github.com/davidjerleke/embla-carousel/assets/2559840/5ba3bf18-5b8f-4a10-8578-8fd0f9edc278

And here’s the CodeSandbox where you can see the current setup:
https://codesandbox.io/p/sandbox/delicate-butterfly-my2x7k

Any help or suggestions would be highly appreciated. Thank you!

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/sandbox/delicate-butterfly-my2x7k

**Comment by davidjerleke** [comment]
@zlateska thanks for your question. This is most likely a duplicate of:

- #579
